bft

```diff
@@ -33,7 +33,7 @@ def main():
     import devices
     from termcolor import colored
     from library import print_bold
-    from devices import board_decider, debian, logstash, elasticlogger
+    from devices import board_decider, debian_decider, logstash, elasticlogger
```

Comment:
debian_decider is used for deciding whether to use DebianBox or NonRootDebianBox based upon the username. I based it on the design for board_decider and similar.